content/en/reference/deployment/config-file/artifacts.md

```diff
@@ -0,0 +1,250 @@
+---
+title: Artifacts and Provider Options (AWS Lambda)
+description: >
+  Declare your AWS Lambda artifacts (functionName, path, type) and provider options.
+---
+
+**AWS Lambda Only**
+
+## Artifacts
+
+This section defines the Lambda artifacts you are deploying. By default, the artifacts reach all targets, but you can specify certain targets if needed.
```

Comment:
we should probably delete the second sentence until https://armory.atlassian.net/browse/CDAAS-3007 is implemented, and instead say 'The artifacts will reach all target for which their is a provider option block for that function name'. (hopefully this gets fixed next sprint, but I don't expect it before GA)